src/20250623_AaveV3Ethereum_AdditionOfUSDeToEthenaPrincipalTokenStablecoinEModes/config.ts

```diff
@@ -0,0 +1,39 @@
+import {ConfigFile} from '../../generator/types';
+export const config: ConfigFile = {
+  rootOptions: {
+    configFile:
+      'src/20250623_AaveV3Ethereum_AdditionOfUSDeToEthenaPrincipalTokenStablecoinEModes/config.ts',
+    force: true,
+    update: true,
+    author: 'Aave-Chan Initiative',
+    pools: ['AaveV3Ethereum'],
+    title: 'Addition of USDe to Ethena Principal Token Stablecoin E-Modes',
+    shortName: 'AdditionOfUSDeToEthenaPrincipalTokenStablecoinEModes',
+    date: '20250623',
+    discussion: 'Addition of USDe to Ethena Principal Token Stablecoin E-Modes',
```

Comment:
Change it to: https://governance.aave.com/t/arfc-addition-of-usde-to-ethena-principal-token-stablecoin-e-modes/22355